### DISCUSSION TITLE
Visual Studio Go To Definition (F12). How to go to my code instead of Generated code

### DISCUSSION THREAD

**Discussion by gregvis** [CommunityToolkit/dotnet#503] [General]
I use F12 quite a bit to jump around my code, but with MVVM generators, it doesn't go where I want it to go

Example

```
[ObservableProperty]
string myProperty


MyProperty = "example"
```

if i go to definition (F12) for MyProperty, it takes me to the generated g.cs file instead of to my code (which should go to "myProperty" . Is there a way to fix this in Visual Studio?  

Same example for in xaml

If I have the command

```
[RelayCommand]
void DoStuff() {}
```

and in xaml I reference it
```
<Button Command={Binding DoStuffCommand} />
```
When I click F12 on DoStuffCommand, I want it to go to DoStuff instead of the g.cs file

**Comment by Sergio0694** [comment]
I'm not aware of a way to do this, but you could open a discussion about this in the Roslyn repo 🙂
https://github.com/dotnet/roslyn
